docs/pages/workflow/expo-cli.mdx

```diff
@@ -348,6 +348,7 @@ From here, you can choose to generate basic project files like:
 
 ## Environment Variables
 
+- `HTTP_PROXY` (**string**) HTTP/HTTPS proxy URL to connect to for all network requests. Configures [https-proxy-agent](https://www.npmjs.com/package/https-proxy-agent).
```

Comment:
```suggestion
- `HTTP_PROXY` (**string**) HTTP/HTTPS proxy URL to connect for all network requests. Configures [https-proxy-agent](https://www.npmjs.com/package/https-proxy-agent).
```

From docs perspective, the change looks good. Just a small suggestion to make the sentence easier to read.